modules/virt-golden-images-namespace-web.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-rh-images-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-golden-images-namespace-web_{context}"]
+= Configuring a custom namespace for golden images by using the web console
+
+You can configure a custom namespace for golden images in your cluster by using the {product-title} web console.
+
```

Comment:
Do the prereqs above need listed here? Maybe not, since you're specifying the namespace here. But maybe the other one does? (I'm assuming it does not, either).